server/graphql/v1/types.js

```diff
@@ -1502,9 +1508,9 @@ export const OrderType = new GraphQLObjectType({
       },
       data: {
         type: GraphQLJSON,
-        description: 'Additional information on order: tax',
+        description: 'Additional information on order: tax and custom fields',
         resolve(order) {
-          return pick(order.data, ['tax']) || null;
```

Comment:
Why remove that, wasn't it a good practice of picking only what we want to return?

Reply:
Oh that was my bad. Will fix it.